pkg/api/static/static.go

```diff
@@ -154,6 +155,10 @@ func staticHandler(ctx *macaron.Context, log *log.Logger, opt StaticOptions) boo
 			if !strings.HasPrefix(path, "/") {
 				// Disambiguate that it's a path relative to this server
 				path = fmt.Sprintf("/%s", path)
+			} else {
+				// A string starting with // or /\ is interpreted by browsers as a URL, and not a server relative path
+				rePrefix := regexp.MustCompile(`^/+|^/\\`)
+				path = rePrefix.ReplaceAll(path, "/")
```

Comment:
@aknuds1 Maybe I am missing something, but check this out https://golang.org/src/regexp/regexp.go?s=20809:20862#L659

Seems like `regexp.ReplaceAll(..)` takes bytes as arguments?

Reply:
Yeah - tests failed too, maybe you thought it was `string.ReplaceAll(...)` instead

Reply:
@dsotirakis yeah I'm fixing :)

Reply:
@aknuds1 Also I think that the way is is now, if you have a string `/\url` it will return `/\url` again, since it will match the first `/`, then escape and replace it with `/` which is the same thing?

Reply:
@dsotirakis I'm in the middle of debugging the regexp, noticed it doesn't catch `/\`.

Reply:
@dsotirakis should be fixed now.